docker/podman-docker.csh

```diff
@@ -0,0 +1,10 @@
+# DOCKER_HOST initialization
+
+if ($?DOCKER_HOST) exit
```

Comment:
Late followup: I was concerned about this `exit`, wondering if it would exit the actual shell. It does not. 
```console
# cat foo.csh
echo aaa
source podman-docker.csh
echo bbb
# source foo.csh
aaa
bbb
```